### PR TITLE
Null Problem issue fix

### PR DESCRIPTION
**Pull request by tmdroid**
This is the fix to the Null Problem issue bug

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -6,6 +6,8 @@
           , src = Url.queryString("link") || "#"
           ;
 
+	title = (title != null) ? title : "Anything";
```
Comment (IonicaBizau) [inline js/main.js]: This should be checked [here](https://github.com/IonicaBizau/anything-for-dummies/blob/gh-pages/js/main.js#L27-L28):

``` js
Url.updateSearchParam("title", prompt("Title") || "Anything");
Url.updateSearchParam("link", prompt("link") || "#");
```

**Comment by IonicaBizau** [comment]
@csanonymus Any updates on this?

**Comment by tmdroid** [comment]
Oh, sorry... Totally forgot about it. I'll update my pull request soon

**Comment by IonicaBizau** [comment]
:+1:

**Comment by tmdroid** [comment]
Check it out. I hope it's better now! :)

File: js/main.js [hunk 2]
```diff
@@ -24,8 +23,8 @@
           ;
 
         if (hash === "create") {
-            Url.updateSearchParam("title", prompt("Title"));
-            Url.updateSearchParam("link", prompt("link"));
+            Url.updateSearchParam("title", prompt("Title") || "Anything");
```
Comment (IonicaBizau) [inline js/main.js:26]: :+1:

**Comment by IonicaBizau** [comment]
LGTM! :+1: 

Add yourself as contributor in `package.json` (create the `contributors` array).

**Comment by IonicaBizau** [comment]
Merged into 1.1.0. Thanks @csanonymus and Happy Birthday! :cake: :+1:

**Comment by tmdroid** [comment]
Thank you! :) :D

**Pull request merged**